client/pom.xml

```diff
@@ -45,6 +45,26 @@
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-client</artifactId>
         </dependency>
+
+        <!-- CXF still using jakarta.ws.rs-api 2.x, we have here 3.X  e.g. package javax.ws.rs.  -->
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-rs-client</artifactId>
+        </dependency>
+        <!--  Import JAX-RS 2.0/javax.ws.rs. from different artifactId  -->
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.1.1</version>
```

Comment:
It would be better to have this dependency defined at the parent pom within the dependency management section and the version specified within the properties section just like how you did it for cxf-rt-rs-client. In that way you can drop the version here.

Reply:
I would not argue with this .  It is here just to discuss the 3 version  JAX-RS api in class path see comments above.